build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -331,3 +331,26 @@ multicluster:
 # The default is antrea-agent's Namespace.
   namespace: {{ .namespace | quote }}
 {{- end }}
+
+secondaryNetwork:
+# Secondary network OVS configuration
```

Comment:
This comment should be moved to before `ovs` in line 339?

Reply:
Aligned it.